package.json

```diff
@@ -18,14 +17,15 @@
   "license": "MIT",
   "devDependencies": {
     "coffee-script": "^1.9.1",
-    "docha": "thompsnm/docha#36f8d21",
+    "cucumber": "^0.5.0",
+    "docha": "^0.1.2",
     "grunt": "^0.4.5",
     "grunt-cli": "^0.1.13",
     "grunt-coffeelint": "^0.0.13",
     "grunt-contrib-coffee": "^0.13.0",
     "grunt-contrib-connect": "^0.9.0",
     "grunt-contrib-requirejs": "^0.4.4",
```

Comment:
Yep, it's used to load `js-fixtures` into the test app that the integration tests run against. See [here](https://github.com/ReadyTalk/cukefarm/blob/master/spec/test_app/scripts/app.js#L1). It's arguably a heavyweight solution for a lightweight problem, but that way all dependencies in the project are managed through npm.